db/seeds.rb

```diff
@@ -25,8 +25,13 @@ def print_seed_info(text, indent: 0, colour: nil, blank_lines_before: 0)
   priority_seeds.index(filename.chomp(".rb")) || Float::INFINITY
 end
 
-seed_files.each do |seed_file|
-  puts "\r\n🪴 planting #{seed_file}"
+DeclarativeUpdates.skip do
```

Comment:
I think we'll just want to `skip(:metadata)` here and include `touch` updates (as we have no way to refresh these outside of the callbacks as it stands)